### ISSUE TITLE
LazySizes and Bgset plugin vs small accessibility issue

### ISSUE THREAD

**Issue by stephsavoie** [aFarkas/lazysizes#200]
I'm using Lazysizes with BGSet plugin and the solution works really great so far.  However, I've found that the Bgset plugin would create a PICTURE tag and, inside it, an IMG tag that has no alt attribute.  This  has no incidence on the normal workflow and the PICTURE tag is in [display: none] so this is not a real big issue, but this makes cry some accessibility checker apps.  Is there a way to add  the missing attribute one way or another?   Easy way I have thought is to inject the attribute by Javascript but It's not necessary my preferred solution...

**Comment by aFarkas** [comment]
Please close if this fixes your issue.

**Comment by stephsavoie** [comment]
It work.  Thanks!

2015-11-27 7:33 GMT-05:00 Alexander Farkas <notifications@github.com>:
